src/api.cpp

```diff
@@ -430,6 +430,7 @@ cl_int CLVK_API_CALL clGetDeviceInfo(cl_device_id dev,
     cl_device_device_enqueue_capabilities val_dev_enqueue_caps;
     cl_device_pci_bus_info_khr val_pci_bus_info;
     cl_device_atomic_capabilities val_atomic_capabilities;
+    std::vector<size_t> subgroup_sizes;
```

Comment:
```suggestion
    std::vector<size_t> val_subgroup_sizes;
```

For consistency with other definitions.